### PR TITLE
Remove react-select

### PR DESCRIPTION
**Pull request by defunctzombie**
Update rosout panel to use fluentui components instead of react-select.

### CODE REVIEW

File: app/panels/Rosout/LogMessage.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import LogMessage from "./LogMessage";
+
+export default {
+  title: "panels/Rosout/LogMessage",
```
Comment (defunctzombie) [inline app/panels/Rosout/LogMessage.stories.tsx:8]: This format for story titles creates a subheading and folder structure within storybook helping to organize the stories.

File: app/panels/Rosout/LogMessage.tsx [hunk 2]
```diff
@@ -33,20 +33,29 @@ function Stamp(props: { stamp: Time }) {
   );
 }
 
+const classes = mergeStyleSets({
```
Comment (defunctzombie) [inline app/panels/Rosout/LogMessage.tsx:36]: @jtbandes is this the correct way to use merge styles?

File: app/theme/ThemeProvider.tsx [hunk 3]
```diff
@@ -15,6 +15,7 @@ const icons: {
   [N in RegisteredIconNames]: React.ReactElement;
 } = {
   Add: <Icons.AddIcon />,
+  Cancel: <Icons.CancelIcon />,
```
Comment (defunctzombie) [inline app/theme/ThemeProvider.tsx:18]: Until I added this the `x` icons didn't show up - it was kind of non-obvious which icon name I needed to add without looking at the fluent source code :/

Comment (jtbandes) [reply app/theme/ThemeProvider.tsx:18]: Yep, I dunno if there’s any way to make this better unless we just go ahead and bundle/register *all* the icons.

The missing icon name should’ve been printed as a console warning though.

File: app/panels/Rosout/LogMessage.stories.tsx (hunk 1 above)
Comment (jtbandes) [reply app/panels/Rosout/LogMessage.stories.tsx:8]: Maybe we should make one big PR to reorganize all our stories? Our code has sort of a convention of including `<>` around the name but I don’t think there’s a good reason for it; we could remove those.

**Pull request merged**